RELEASE.md

```diff
@@ -49,20 +51,40 @@ version number for the new release. We are following
   git tag -s v<version>
   ```
 
-* Update version in [CMakeLists.tx](https://github.com/greenbone/gsa/blob/master/CMakeLists.txt)
-
-* Create a commit
+* Push changes and tag to GitHub
 
   ```sh
-  git commit -m "Update version after <version> release"
+  git push --tags upstream
   ```
 
-* Push changes and tag to Github
-
+* Create a PR for your changes and get it reviewed and merged into the branch you want to release
+
+Now you need to create a GitHub release. This will include the actual release and adding signatures and signed binaries of the source code and node modules.
+
+* Create GitHub release  
+  * Navigate to the main page of the repository
+  * Click on "Releases"
+  * Click "Draft a new release"
+  * Type the version number you want to release in the "Version Tag" input. We are using calendar versioning starting with a v, e.g. `v21.4.1`.
+  * Select the branch you want to release in the "Target:" input
+  * As title add `Greenbone Security Assistant (GSA) <version>`
+  * Copy the relevant [CHANGELOG.md](https://github.com/greenbone/gsa/blob/gsa-21.04/CHANGELOG.md) entries into the description
+  * If you are ready to release, click on "Publish release", otherwise you can save it as a draft.
+* Sign the release
+  * Go to the "Releases" section on GitHub and download the source code tarballs created automatically by GitHub
+  * Go to your source directory in your development environment and compress the `node_modules` as tar.gz and .zip. If you change branches a lot, before you compress the directory, you might want to make sure that the node modules are actually correct by deleting the folder and creating it again by running `yarn`.
+  * Make sure that the source code tarballs contain the same data as the git repository
+  * Sign _all four_ files with your key
   ```sh
-  git push --tags upstream master
+  gpg --default-key 0ED1E580 --detach-sign -a xyz.tar.gz
+  gpg --default-key 0ED1E580 --detach-sign -a xyz.zip
   ```
-
-* Create a Github release
-
-  See https://help.github.com/articles/creating-releases/
+  This will create files with an .asc file type. If .sig files were created, you probably didn't use the option `--armor` or `-a`.
+  * Rename the node modules files and their signatures with the approriate version into, e.g. `gsa-node-modules-<version>.tar.gz`.
+  * Edit your release again and add the following files as binary
+    * the .tar.gz and .zip files of the node modules
+    * both of their detached signatures
+    * both detached signatures of the source code tarballs
+  * Click "Update release"
+
```

Comment:
```suggestion

## Cleanup after the release

* Create a new, empty unreleased section in the `CHANGELOG.md` file
* Update the version in _all_ version files:
  * increment the version in the `CMakeLists.txt` files, and set `dev` version again
  * increment the version in `gsa/src/version.js` and `gsa/package.json`

```

Reply:
Incrementing the versions here again will make the previous steps with these files incorrect.

Reply:
Yeah but this is the correct place to increment the versions.
Technically the versions should be incremented here, with a `dev` version. And Before the release only the `dev` should be removed.

Reply:
Yeah I would remove it in the top of the page and increment and set dev *after*  release. 
Than only remove the dev thingies before release.

The About page line is also obsolete now, since we use the `gsa/src/version.js` file now.

Reply:
That line is not in the PR. Are you sure you are reviewing the correct version?